runtime/lua/vim/diagnostic.lua

```diff
@@ -1045,26 +1060,6 @@ function M._get_virt_text_chunks(line_diags, opts)
   end
 end
 
---- Callback scheduled when leaving Insert mode.
----
---- This function must be exported publicly so that it is available to be
---- called from the Vimscript autocommand.
----
---- See @ref schedule_display()
----
----@private
-function M._execute_scheduled_display(namespace, bufnr)
```

Comment:
Keep the public function and have it simply call the new local function, and add a call to `vim.deprecate` that indicates that this function will be removed in the 0.9 release.